app/src/main/java/com/ciandt/dragonfly/example/data/remote/RemoteProjectListener.kt

```diff
@@ -19,7 +19,12 @@ class RemoteProjectListener(private val localDataSource: LocalDataSource) : Chil
     }
 
     override fun onChildChanged(dataSnapshot: DataSnapshot, prevChildKey: String?) {
-        onChildAdded(dataSnapshot, prevChildKey)
+        val project = DataSnapshotToProjectEntityMapper(dataSnapshot).map()
+        project?.let {
+            runOnBackgroundThread {
+                localDataSource.update(it)
+            }
+        }
     }
```

Comment:
When a project already exists with one version in db and it is deleted, we didnt delete the version from database. Now we do it.